account_invoice_supplier_ref_unique/wizards/account_invoice_refund.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+<data>
+
+    <record model="ir.ui.view" id="account_invoice_refund_form_view">
+        <field name="name">account.invoice.refund.form (in account_invoice_supplier_ref_unique)</field>
```

Comment:
OK, no problem. This is just a matter of taste, although not putting it is one thing less to translate.